src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -122,6 +127,8 @@ export class ExamParticipationComponent implements OnInit, OnDestroy {
             // init studentExam and activeExercise
             this.studentExam = studentExam;
             this.activeExercise = studentExam.exercises[0];
+            // set endDate with workingTime
+            this.individualStudentEndDate = this.exam.startDate ? moment(this.exam.startDate).add(studentExam.workingTime, 'seconds') : this.individualStudentEndDate;
```

Comment:
Here again, if start date is not defined, we set the end date to now? Does this make sense?

Reply:
See my answer to your other comment. And also it would be first set to the default end date of the exam and then only to now if this isn't available. And I can't really image that this would.